specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-05-01-preview/Common.json

```diff
@@ -1,80 +0,0 @@
-{
```

Comment:
This file cannot be deleted from 2019-05-01-preview. It's still being referenced in the 2019-05-preview tag. And it needs to be in the same folder as swaggers that reference it.